nuxeo-coldstorage/src/main/resources/OSGI-INF/coldstorage-events-contrib.xml

```diff
@@ -22,6 +22,12 @@
               class="org.nuxeo.coldstorage.events.DownloadColdDocumentListener">
       <event>download</event>
     </listener>
+    <!-- UpdateThumbnailListener has a priority of 999, more allows to be evaluated later -->
+    <listener name="checkAlreadyInColdStorageListener" priority="900"
+      class="org.nuxeo.coldstorage.events.CheckAlreadyInColdStorageListener">
+      <event>beforeDocumentModification</event>
+      <event>documentCreated</event>
```

Comment:
why we filter on documentCreated and documentUpdated in the code and here we ask for beforeDocumentModification ?